keras_nlp/models/distilbert/distilbert_models.py

```diff
@@ -62,7 +63,7 @@ class DistilBertCustom(keras.Model):
     Examples:
     ```python
     # Randomly initialized DistilBERT encoder
-    model = keras_nlp.models.DistilBertCustom(
+    model = keras_nlp.models.DistilBert(
```

Comment:
FYI I'm not sure the architecture outlined here pertains to any DistilBERT arch in use (e.g., [base](https://huggingface.co/distilbert-base-uncased/blob/main/config.json)). Not sure if this matters.